Polls/ViewControllers/QuestionListViewController.swift

```diff
@@ -26,6 +26,20 @@ class QuestionListViewController : UITableViewController, QuestionDetailViewCont
     loadData()
   }
 
+  override func viewWillAppear(animated: Bool) {
+    super.viewWillAppear(animated)
+
+    if let selectedIndex = tableView.indexPathForSelectedRow() {
```

Comment:
but yet it was sometimes frustrating to trying to figure out things like this :memo: :thought_balloon: